osm.py

```diff
@@ -401,6 +417,8 @@ def main():
             print("Error: Cannot locate the 'diff' command, aborting.")
             exit(-1)
 
+    load_osm_config()
```

Comment:
Yeah, I know, I forgot the args.config parameter here, it will be fixed in a later commit :-)